source/2019-12-02-countdown-to-the-new-year-ember-modifier.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Countdown to The New Year- Ember Modifier
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+responsive: true
+---
+
+## Day 3
+
+This is the third in our DecEmber series– _"Countdown to The New Year: 31 Days of Ember Addons"_. We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!
+
+Today we are going to look at a rising star in our addon ecosystem- [ember-modifier](https://emberobserver.com/addons/ember-modifier). 
+
+READMORE
+
+### What It Does
+
+The `ember-modifier` addon provides an API for authoring element modifiers in Ember. It mirrors Ember's [helper API](https://api.emberjs.com/ember/3.14/classes/Ember.Templates.helpers), and has forms for writing both simple functional modifiers and more complicated class modifiers.
```

Comment:
It could be nice to add an example here. Unless someone is using Octane, they don't know what a modifier is or what kinds of problems it solves.

Reply:
A modifier can add special, automatic behavior to a DOM element, such as tracking scroll position or adding new event listeners. After you write a modifier, you can use it in a template like this:

```
<div {{my-modifier}}>
  Some more content
</div>
```

 Developers who have tried out the Octane preview might already be familiar with [ember-render-modifiers](https://github.com/emberjs/ember-render-modifiers), which provide some modifiers like `{{did-insert}}` out of the box. With ember-modifiers, you can go beyond the box to build whatever your app needs!

Reply:
Great idea, thank you @jenweber!